get-customer-by-org-number/src/main/java/no/unit/nva/customer/get/GetCustomerByOrgNumberHandler.java

```diff
@@ -0,0 +1,70 @@
+package no.unit.nva.customer.get;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.customer.ObjectMapperConfig;
+import no.unit.nva.customer.exception.InputException;
+import no.unit.nva.customer.model.Customer;
+import no.unit.nva.customer.service.CustomerService;
+import no.unit.nva.customer.service.impl.DynamoDBCustomerService;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.handlers.ApiGatewayHandler;
+import nva.commons.handlers.RequestInfo;
+import nva.commons.utils.Environment;
+import nva.commons.utils.JacocoGenerated;
+import nva.commons.utils.RequestUtils;
+import org.apache.http.HttpStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GetCustomerByOrgNumberHandler extends ApiGatewayHandler<Void, CustomerIdentifier> {
+
+    public static final String ORG_NUMBER = "orgNumber";
+
+    private final CustomerService customerService;
+    private static final Logger logger = LoggerFactory.getLogger(GetCustomerByOrgNumberHandler.class);
+
+    /**
+     * Default Constructor for GetCustomerHandler.
+     */
+    @JacocoGenerated
+    public GetCustomerByOrgNumberHandler() {
+        this(new DynamoDBCustomerService(
+            AmazonDynamoDBClientBuilder.defaultClient(),
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If you don’t explicitly set an AWS region, the SDK consults the default region provider chain to try and determine the region to use.  However, the resulting delay might lead to a poor cold-start. If code portability is not an issue, consider explicitly setting the AWS region using the *withRegion()* method.

[Learn more](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-region-selection.html#automatically-determine-the-aws-region-from-the-environment)